bibliograph/core/utils.py

```diff
@@ -221,16 +261,19 @@ def bin_search(binary, default=_marker):
 
         We assume we have a python(.exe) installed anywhere in
         the path. This seems one of the safest test.
+
+        >>> import pretext; pretext.activate()
         >>> from bibliograph.core.utils import bin_search
-        >>> bin_search('python') != ''
+        >>> bin_search('python') != ''  # doctest: +SKIP
```

Comment:
I've put up a patch for that problem. https://github.com/moreati/b-prefix-all-the-doctests/pull/2